packages/recommend-vdom/src/DefaultHeader.tsx

```diff
@@ -5,6 +5,10 @@ import { cx } from './utils';
 
 export function createDefaultHeaderComponent({ createElement }: Renderer) {
   return function DefaultHeader<TObject>(props: ComponentProps<TObject>) {
+    if (!props.recommendations || props.recommendations.length < 1) {
```

Comment:
`props.recommendations` is always an array so why this first guard? We can also merge it with the following condition?

```suggestion
    if (props.recommendations.length === 0 || !props.translations.title) {
```